### DISCUSSION TITLE
How set spacedrive to be  default exploer in windows?

### DISCUSSION THREAD

**Discussion by wwalkingg** [spacedriveapp/spacedrive#1594] [Community Support]
windows11 can change default explorer in Setting -> App -> Default App -> Resource Explorer.
![image](https://github.com/spacedriveapp/spacedrive/assets/42681996/4b3066aa-b9d0-48cd-8995-708e14136f7e)
But no SpaceDrive option.
![image](https://github.com/spacedriveapp/spacedrive/assets/42681996/5dc0f64e-beb9-4a75-a085-96b8cf9661a5)

**Comment by headquarter8302** [comment]
I think this feature only works for applications that has explicit tie-ins with the respective APIs in Windows, which I don't think Spacedrive supports at the moment. Plus, those are only for Burn, erase disc, and zip operations in Explorer, not the default app by itself.

You can _probably_ work around this by editing the registry:
* https://superuser.com/a/1644099
* https://superuser.com/a/241472

**However**, I do not think Spacedrive even supports this, and you may cripple your Explorer UI by doing this unless you have backed up that specific reg key
